integration/docker/entrypoint.sh

```diff
@@ -12,8 +12,9 @@
 
 set -e
 
+expected='"formatMaster", "formatWorker", "launchMaster", "launchWorker", "master, "worker", or "proxy"'
```

Comment:
this set of commands would be hard for a user to navigate - it's confusing what's the difference between `launchMaster` vs `master` vs `formatMaster`. Perhaps we could control formatting with a flag instead?

Reply:
sg